src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const object = sourceString
```

Comment:
The variable name 'object' is not descriptive and can be confusing as it's a term used in JavaScript for object data types. Consider using a more descriptive name, such as 'styleObject' or 'cssObject', to enhance readability and maintainability of the code.